.github/workflows/build.yml

```diff
@@ -36,6 +36,6 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           push: true
-          tags: ghcr.io/aica-technology/component-example:latest
+          tags: ghcr.io/${{ github.repository }}/template-component-package:latest
```

Comment:
This will make the string `ghcr.io/aica-technology/component-template/template-component-package:latest` (see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#github-context).

We could either do:
 * `ghcr.io/${{ github.repository }}:latest`: assume you want the same image name as the repository
 * `ghcr.io/${{ github.repository_owner }}/component-example:latest`: more flexible but might need replacement from the script?

Reply:
I disagree with that I think. When you use the template option for cloning, the repo name is empty, and the user needs to put in the name itself. So what happens is that this will do `ghcr.io/domire8/intelligent_name/template-component-package:latest` where the `template-component-package` part should and would be replaced through the `rename.sh` script

Reply:
And the script already replaces the files in `.github`, the script help text is just slightly outdated

Reply:
`ghcr.io/domire8/intelligent_name/template-component-package:latest` isn't a valid package name though
Packages are global to your account and follow the Docker registry syntax of `namespace/repo`, what you would need is `ghcr.io/domire8/template-component-package:latest` or more likely `ghcr.io/domire8/intelligent_name:latest`.